examples/todo-typescript/src/client/MainPage.tsx

```diff
@@ -1,17 +1,18 @@
-import './Main.css';
 import React, { useEffect, FormEventHandler, FormEvent } from 'react';
+import waspLogo from './waspLogo.png';
+import './Main.css';
+// Wasp imports 🐝 = }
 import logout from '@wasp/auth/logout.js';
 import useAuth from '@wasp/auth/useAuth.js';
 import { useQuery } from '@wasp/queries'; // Wasp uses a thin wrapper around react-query
 import getTasks from '@wasp/queries/getTasks';
 import createTask from '@wasp/actions/createTask';
 import updateTask from '@wasp/actions/updateTask';
-import waspLogo from './waspLogo.png';
-import { Task } from './types'
+import type { Task } from '@wasp/entities'
 
 export function MainPage() {
   const { data: user } = useAuth();
-  const { data: tasks, isLoading, error } = useQuery<unknown, Task[]>(getTasks);
+  const { data: tasks, isLoading, error } = useQuery(getTasks);
```

Comment:
Let's use our super cool user prop here:
```typescript
import User from "@wasp/auth/types.js";

// ...

export const MainPage = ({ user }: { user: User }) => {
  const { data: tasks, isLoading, error } = useQuery(getTasks);
  // ...
}
```

Reply:
One small caveat: the `User` type might not be available _just yet_. I need to merge a PR that introduces it.